components/Footer/index.js

```diff
@@ -6,6 +6,7 @@ export default function Footer() {
   return (
     <footer className={Styles.footer}>
       <p className={Styles.footer__copyright}>&copy; unbreak.tech {date.getFullYear()}</p>
+      <a href="https://github.com/whitep4nth3r/unbreak.tech" target="_blank" rel="noopener">Contribute</a>
```

Comment:
whoops! I actually only styled the visited link by accident 🙈  I was rushing because I wanted to submit it by the end of your stream.

Also I made the footer items be on top of each other in mobile and next to each other on desktop 👍